requirements.txt

```diff
@@ -17,3 +17,4 @@ jupytext
 
 # Notebook requirements
 pygraphviz
+basemap
```

Comment:
According to the [docs](https://matplotlib.org/basemap/users/intro.html#deprecation-notice), `basemap` is intended to be replaced by `cartopy`... Should we try to use cartopy instead? FWIW I tried installing basemap on an M1 linux machine and was not able to do so as there are no m1 wheels.